tests/test_csc.py

```diff
@@ -342,7 +341,7 @@ async def test_auto_exposure(self):
 
             # Set the auto exposure time interval to a low value so
             # the test doesn't take so long
-            self.csc.config.autoExposureInterval = 2.0
+            self.csc.config.auto_exposure_interval = 2.0
```

Comment:
You might also consider setting this in tests/data/config/_init.yaml. It's a good place to set useful-for-testing parameters. Mind you, what you've done is also fine, but it does require knowing more about the internals of the class.

Reply:
I'll leave it as is.